cfgov/unprocessed/js/modules/util/js-loader.js

```diff
@@ -12,29 +12,12 @@ function loadScript( url, callback ) {
   const script = document.createElement( 'script' );
   script.type = 'text/javascript';
 
-  // Handle Internet Explorer
-  if ( script.readyState ) {
-    script.onreadystatechange = function() {
-      if (
-        script.readyState === 'loaded' ||
-        script.readyState === 'complete'
-      ) {
-        script.onreadystatechange = null;
-        if ( callback ) {
-          return callback();
-        }
-      }
-      return null;
-    };
-  // Other browsers.
-  } else {
-    script.onload = function() {
```

Comment:
This is the support table for script.onload http://kangax.github.io/jstests/script-element-onload-attribute-support/